WalletWasabi.Fluent/Views/Wallets/Receive/ReceiveAddressView.axaml

```diff
@@ -35,9 +35,10 @@
             <Button Margin="5 0 0 0" ToolTip.Tip="Show on the hardware wallet" DockPanel.Dock="Right" Classes="plain" Command="{Binding ShowOnHwWalletCommand}" IsVisible="{Binding IsHardwareWallet}">
               <PathIcon Data="{StaticResource share_regular}"/>
             </Button>
```

Comment:
Can you change this to an `AnimatedButton` too? `copy_confirm` as ClickIcon is good here too.

Reply:
Sure I will make it animated button too.

Reply:
Added